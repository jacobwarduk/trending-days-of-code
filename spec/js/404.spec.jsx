import React from 'react';
import {shallow} from 'enzyme';
import FourOhFour from '../../js/404';

describe('FourOhFour', () => {
	'use strict';
	describe('when rendered', () => {
		let component = shallow(<FourOhFour />);

		it('should render correctly', () => {
			expect(component).toMatchSnapshot();
		});
	});
});
