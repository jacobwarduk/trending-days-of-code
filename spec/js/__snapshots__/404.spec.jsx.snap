// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour when rendered should render correctly 1`] = `
<h1>
  404 PAGE NOT FOUND
</h1>
`;
